src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -12,92 +12,187 @@ This page quickly describes how [Issues](https://guides.github.com/features/issu
 
 #### Regressions
 
 If the bug report is confirmed, one of the key elements to evaluate is whether it is a regression. A regression is a bug that cannot be observed in the previous PrestaShop version, it means the software quality level has decreased instead of increasing. Regressions are usually milestoned to be fixed in the next patch version.
 
-#### This is not easy
+### Support request
 
-The work of analyzing and testing all submitted bug reports is a very complex one, because there might be a very diverse range of reports. Moreover quite a huge number of them are actually not related to the software but to how the shop is being used: the server configuration, the shop configuration, the installed modules and the installed theme might introduce buggy behaviors that the user mistakenly believes come from the software.
+Sometimes support requests are submitted on GitHub, ranging from questions about the software to "please help me to do X in my shop". GitHub issues are not a Q&A forum, so these requests are usually redirected to other channels using the [Support template](https://github.com/PrestaShop/PrestaShop/issues/new?template=3_support_question.md) and closed.
 
-This is why so much Issues cannot be reproducted on our side, but to find it out multiple explorations and attempts are necessary.
+### Other
 
-### Support request
+Some issues do not fit in the previous categories, such as [Releases Issues](https://github.com/PrestaShop/PrestaShop/labels/Release) which are used to follow the progress of a release, or [Epic Issues](https://github.com/PrestaShop/PrestaShop/labels/Epic) that group together all requests belonging to the same subject.
 
-We sometimes receive support requests on GitHub, ranging from questions about the software to "please help me to do X in my shop" requests. GitHub is for the software development so we usually redirect users to other channels using the [Support template](https://github.com/PrestaShop/PrestaShop/issues/new?template=3_support_question.md).
+## Projects
 
-### Other
+Available in the "[Projects](https://github.com/PrestaShop/PrestaShop/projects)" tab of PrestaShop’s GitHub repository, versions kanban boards define the scope of a minor version in terms of what the Core Maintainers and Core Product teams agree must be included before that release is out.
+Both the product and the maintainers teams use it to prepare and plan the next developments and, since we deal with an open source project, the purpose of having those kanbans public is to help the community and the external mainteners to know what is at stake.
+
+### Patches versions
+
+There should only have regressions issues in the kanban of a patch version. For example, 1.7.7.1 and 1.7.7.2 (and so on...) kanbans only includes the regressions of the 1.7.7.0 version.
```

Comment:
Maybe mention that sometimes we also include critical or impactful fixes when the timing is important?

For example I remember we added the fix for https://github.com/PrestaShop/PrestaShop/issues/22430 because it was considered an important issue.

Also we dont mention vulnerabilities fixes.

Reply:
I added security & critical issues